### ISSUE TITLE
Proposal grouping

### ISSUE THREAD

**Issue by codehag** [tc39/tc39.github.io#61]
There was some discussion at yesterday's meeting about proposal grouping. For example, classes proposals really work together and do not make as much sense separately. This issue is here for discussion of this topic.

Some idea:
- dedicated pages
- tagging grouped proposals with for example `proposal-group--class` or something similar and displaying those together

**Comment by littledan** [comment]
We may also want to have some dedicated text for proposal groups. Not sure where the data should live.

To give another example, we could have a proposal group for RegExp features.

**Comment by rkirsling** [comment]
Yeah, by my earlier comment, I was imagining that proposal READMEs (of a sufficient stage) would be updated to include any applicable grouping. I actually think this could be useful for everyone involved.
